packages/validator/src/rules/no-undefined-path-parameters/no-undefined-path-parameters.ts

```diff
@@ -0,0 +1,56 @@
+import { constructHttpPath } from "@fern-api/ir-generator";
+import chalk from "chalk";
+import { Rule, RuleViolation } from "../../Rule";
+
+export const NoUndefinedPathParametersRule: Rule = {
+    name: "no-duplicate-enum-values",
```

Comment:
wrong name - we can get rid of this field tbh. actually the benefit of keeping it is if the rule throws, we can catch and be like `rule <xyz> threw an error: <error>`